projects/igniteui-angular/src/lib/date-picker/date-picker.component.ts

```diff
@@ -161,7 +163,15 @@ export class IgxDatePickerComponent implements IDatePicker, ControlValueAccessor
      * ```html
      * <igx-date-picker [label]="Calendar"></igx-date-picker>
      * ```
+     * @deprecated By default the label is set to 'Date'. Use
+     * ````html
+     * <igx-date-picker>
+     *      <label igxLabel>Custom label</label>
+     * </igx-date-picker>
+     * ````
+     * to set a custom label.
      */
+    @DeprecateProperty(`By default the label is set to 'Date'.`)
```

Comment:
Same here - this is a console warning, really doesn't tell me much that the default value is Date :) Consider text similar to my above recommendation - "Use igxLabel.. "